content/blog/2013-06-19-usgs-app.md

```diff
@@ -33,7 +33,6 @@ We think of the USGS app as a visual interface to show what can be do in R on th
 
 [usgsapp]: http://glimmer.rstudio.com/ropensci/usgs_app/
```

Comment:
link broken in preview and in original [usgsapp]: http://glimmer.rstudio.com/ropensci/usgs_app/

Reply:
thanks, this will be tackled in #585